### PR TITLE
Copy over static folder

### LINKED ISSUE #47: Copy over static folder
This issue focuses on copying any static assets the user has whether they are linked or not

For this we need:

- Add static_folder option and default it to "./static"
- Copy over the content of this folder to the dist directory

### PR DESCRIPTION
**Pull request by SaraVieira**
PR copies the static folder recursively

Maybe this functionlity could be implemented in axoasset

closes https://github.com/axodotdev/oranda/issues/47

### CODE REVIEW

File: src/site/mod.rs [hunk 1]
```diff
@@ -8,6 +8,9 @@ mod head;
 mod header;
 mod html;
 pub mod markdown;
+extern crate fs_extra;
```
Comment (ashleygwilliams) [inline src/site/mod.rs]: why is this here? you shouldn't need this (it's a very old and totally vestigal piece of rust...)

File: src/site/mod.rs [hunk 2]
```diff
@@ -8,6 +8,9 @@ mod head;
...
+use fs_extra::copy_items;
+use fs_extra::dir;
```
Comment (ashleygwilliams) [inline src/site/mod.rs]: i would not bring these into scope because it will be confusing where they come from

**Review by ashleygwilliams — APPROVED** [review]
so this seems OK, i do think we'll want to put this in axo asset but it's fine in here for now. a little worried about this dep - but i think for now we can keep it and revisit when we move to axoasset- can you file an issue on axoasset for this?

File: src/site/mod.rs (hunk 1 above)
Comment (SaraVieira) [reply src/site/mod.rs]: you are right, its removed

File: src/site/mod.rs (hunk 2 above)
Comment (SaraVieira) [reply src/site/mod.rs]: also fixed
